Development/src/pages/devices/ChannelMappingMatrix.js

```diff
@@ -48,129 +62,321 @@ const TooltipDivider = withStyles({
     },
 })(Divider);
 
-const getOutputTooltip = (outputId, outputItem, io) => {
+const InteractiveTooltip = ({
+    getTooltip,
+    display,
```

Comment:
Rather than a `display` function, I think the idiomatic React way would be to take the children and pass them on to the underlying `Tooltip`.

So user would write:
```
<InteractiveTooltip ...>
    <SomeComponent />
    <AnotherComponent />
</InteractiveTooltip>
```

And `InteractiveTooltip` would use [`props.children`](https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891).